entrypoint.sh

```diff
@@ -0,0 +1,38 @@
+#!/bin/sh -l
+
+git_setup() {
+  cat <<- EOF > $HOME/.netrc
+		machine github.com
+		login $GITHUB_ACTOR
+		password $GITHUB_TOKEN
+		machine api.github.com
+		login $GITHUB_ACTOR
+		password $GITHUB_TOKEN
+EOF
+  chmod 600 $HOME/.netrc
+
+  git config --global user.email "$GITHUB_ACTOR@users.noreply.github.com"
+  git config --global user.name "$GITHUB_ACTOR"
+
+  : ${INPUT_PUSH_BRANCH:=`echo "$GITHUB_REF" | awk -F / '{ print $3 }' `}
+}
+
+git_setup
+
+# debug
```

Comment:
You can probably remove that debug code now.

Reply:
Yeah I guess so

Reply:
Removed!